### PR TITLE
Update rules.txt

### PR DESCRIPTION
**Pull request by ericwclymer**

### CODE REVIEW

File: rules.txt [hunk 1]
```diff
@@ -22,12 +22,12 @@
...
-    5.3  The winning point must be the result of a lob.
-    5.4  The Fillipini Rule
-      5.4.1  If the team on game point scores a point that is not a lob the opponents shall incur a strike.
-      5.4.2  If the team incurring a strike shall score 1 point,  their strikes shall be reset.
-      5.4.3  If a team incurs 3 strikes they shall loose the game.
-      5.4.4  Members of all teams are required to acknowledge the Fillipini rule upon any team reaching game point by verbally confirming that "The Fillipini rule is in effect".   
+    5.3  The winning point must be the result of a lob, except in the case of 5.4, the Three-Strike (Filippini) Rule.
+    5.4  The Three-Strike (Filippini) Rule
+      5.4.1  If a team is within one point of a winning score and scores a point that is not the result of a lob, their opponents shall incur a strike.
+      5.4.2  If the opponent of a the team within one point of a winning score scores any number of points, their strikes shall be reset.
```
Comment (DDropinski) [inline rules.txt:28]: How about simplifying this to something like `If a team earns at least one (1) point, their strikes shall be reset to zero (0)`.

I can't think of a scenario where that isn't true (and matters).

Comment (andypryor) [inline rules.txt:28]: makes sense to me.  @ericwclymer ?

Comment (ericwclymer) [inline rules.txt:28]: Love it.

File: rules.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@
     2.1.1 The first player to strike the ball on a return is the "setter".  The partner of the setter when finishing the return is the "spiker".  The setter may choose to return the ball directly,  or they may strike the ball so that the "spiker" is able to return the ball to the opponents side of the table.  When the "setter" and "spiker" both hit the ball on a return this is known as a "lob".
     2.1.2  Lob Scoring
       2.1.2.1  A lob that scores immediately after a 1 hit return shall be worth 2 points.
-      2.1.2.2  A lob that scores after a lob from the opponents shall be worth (N+1) points,  where N is the number of consective lobs that have occurred between both teams since a 1 hit return.
+      2.1.2.2  A lob that scores after a lob from the opponents shall be worth (N+1) points,  where N is the number of consective lobs that have occurred between both teams since a 1 hit return. For example, a if the serving team hits a lob at the first legal opportunity, the receiving team fields and converts their own lob, it shall be worth 3 points.
```
Comment (DDropinski) [inline rules.txt:10]: Grammatical error: `For example, a if the`

**Comment by andypryor** [comment]
Applied discussed changes in https://github.com/andypryor/lob-pong/pull/3

**Pull request merged**